### PR TITLE
bpo-30935: update get_event_loop docs

### PR DESCRIPTION
**Pull request by daxlab**
https://bugs.python.org/issue30935

### CODE REVIEW

File: Doc/library/asyncio-eventloops.rst [hunk 1]
```diff
@@ -165,7 +165,7 @@ An event loop policy must implement the following interface:
 
    .. method:: get_event_loop()
 
-      Get the event loop for the current context.
+      Get the event loop for the current coroutine.
```
Comment (cjerdonek) [inline Doc/library/asyncio-eventloops.rst]: You want to say this not in the documentation of the interface, but in the documentation of the default policy.

Also, in the bpo issue I suggested adding a "Changed in 3.6" directive, but I don't see that here.

Comment (daxlab) [reply Doc/library/asyncio-eventloops.rst]: Isn't it already [there](https://github.com/python/cpython/blob/master/Doc/whatsnew/3.6.rst#L821) ?

Comment (cjerdonek) [reply Doc/library/asyncio-eventloops.rst]: In "What's New," yes, but not in the asyncio section of the docs.

Comment (cjerdonek) [reply Doc/library/asyncio-eventloops.rst]: See the documentation of [`AbstractEventLoop.run_forever()`](https://docs.python.org/3/library/asyncio-eventloop.html#asyncio.AbstractEventLoop.run_forever) for an example. In this case, it says, "Changed in version 3.5.1."

**Review by Mariatta — CHANGES_REQUESTED** [review]
Please add the "Changed in 3.6" directive as suggested by @cjerdonek

**Comment by daxlab** [comment]
I didn't expect the Spanish Inquisition!

**Review by cjerdonek — CHANGES_REQUESTED** [review]
This still doesn't update the [`get_event_loop()`](https://docs.python.org/3/library/asyncio-eventloops.html#asyncio.AbstractEventLoopPolicy.get_event_loop) docs as I said in the original bpo issue and in my previous review comment.

**Comment by 1st1** [comment]
Please update the docstring of `get_event_loop()`

**Comment by csabella** [comment]
@daxlab  There were some additional changes requested since your last commit.  Would you be able to take a look at those?  Thanks!

**Review by 1st1 — APPROVED** [review]

**Pull request merged**